.gitignore

```diff
@@ -5,3 +5,8 @@ logs/
 *.downloaded.rb
 *.swp
 *.swo
+
+#eclipse
```

Comment:
@jamesk can you please add these to https://help.github.com/articles/ignoring-files/#create-a-global-gitignore these shouldn't be project specific.